### PR TITLE
Onboarding middleware conflict resolve

### PR DESCRIPTION
**Pull request by poorphd**
## Description

Conflict resolve
https://github.com/Canto-Network/Canto/pull/110

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] targeted the correct branch (see [PR Targeting](https://github.com/canto-network/canto/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [ ] included the necessary unit and integration tests
- [x] reviewed "Files changed" and left comments if necessary
 
### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -751,24 +726,30 @@ func NewCanto(
...
+
+		// canto, ethermint modules
 		evm.NewAppModule(app.EvmKeeper, app.AccountKeeper),
 		epochs.NewAppModule(appCodec, app.EpochsKeeper),
+		inflation.NewAppModule(app.InflationKeeper, app.AccountKeeper, app.StakingKeeper),
 		feemarket.NewAppModule(app.FeeMarketKeeper),
-		coinswap.NewAppModule(appCodec, app.CoinswapKeeper, app.AccountKeeper, app.BankKeeper),
+
+		// TODO: Modules that have not yet been implemented for simulation
+		// govshuttle, csr, recovery, erc20
```
Comment (dongsam) [inline app/app.go:752]: need to add coinswap module, fix comments

**Review by dongsam — CHANGES_REQUESTED** [review]

File: app/app.go (hunk 1 above)
Comment (poorphd) [reply app/app.go]: coinswap module is added https://github.com/b-harvest/Canto/pull/25/commits/ca2307f57fa474d81b39702ed2592db5b096fb90

File: app/app.go [hunk 2]
```diff
@@ -784,16 +765,18 @@ func NewCanto(
 
 	maxGasWanted := cast.ToUint64(appOpts.Get(srvflags.EVMMaxTxGasWanted))
 	options := ante.HandlerOptions{
-		AccountKeeper:   app.AccountKeeper,
-		BankKeeper:      app.BankKeeper,
-		EvmKeeper:       app.EvmKeeper,
+		AccountKeeper: app.AccountKeeper,
+		BankKeeper:    app.BankKeeper,
+		EvmKeeper:     app.EvmKeeper,
+		//StakingKeeper:   app.StakingKeeper,
```
[the remark this answers is not in the document]
Comment (poorphd) [reply app/app.go]: removed

**Comment by poorphd** [comment]
It seems simulation for coinswap module has some issues.

**Review by dongsam — APPROVED** [review]

**Pull request merged**